data/helpers.d/network

```diff
@@ -17,7 +17,7 @@ ynh_find_port () {
 	ynh_handle_getopts_args "$@"
 
 	test -n "$port" || ynh_die --message="The argument of ynh_find_port must be a valid port."
-	while ss -nltu | grep -q -w :$port       # Check if the port is free
+	while ss -nltu | grep -q -w "*:$port"       # Check if the port is free
```

Comment:
```suggestion
	while ss -nltu | grep -q -E ":$port\s+"       # Check if the port is free
```

Reply:
A better way:

```suggestion
	while ss -nltu | awk '{print$5}' | grep -q -E ":$port$"       # Check if the port is free
```

Reply:
Agree with that last proposition. Maybe though, you should escape the first $.

Reply:
You mean `...\$port...` ? I tried, but it's no longer considered a variable.

Reply:
Sorry, forgot the script itself !
I was just reading the line and its regex...